coalib/processes/BearRunning.py

```diff
@@ -1,37 +1,39 @@
+import logging
 import queue
 import traceback
 
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
+from coalib.processes.communication.LogMessage import LOG_LEVEL
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
 from coalib.results.Result import Result
 
 
-def send_msg(message_queue, timeout, log_level, *args, delimiter=' ', end=''):
+def send_msg(log_level, *args, delimiter=' ', end='', message_queue=None,
```

Comment:
do not change the order of parameters.  that breaks the API, and is probably why your builds are broken.

Reply:
@jayvdb but to make its default value `None`, I had to change its order as "non-default argument follows default argument"

Reply:
The reason builds are broken is maybe because of the while loop conditions in `process_queue` method

Reply:
Oh sorry, I understand what I did was wrong here. I'll fix it ;)
>do not change the order of parameters

Reply:
you must not break the API.  Setting a default of `None` is optional, and obviously can not be done in this situation, otherwise you break the API.